gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.9.1-bin.zip
```

Comment:
Did you try 7.2?

Reply:
Yep. Spine 1.x (bootstrap plugin) is not compatible with Gradle 7.x. I plan to migrate the app to Spine 2.x and do the migration to Gradle 7.x there.